build.cake

```diff
@@ -28,13 +28,8 @@ var packageVersion = version + modifier + dbgSuffix;
 // SUPPORTED FRAMEWORKS
 //////////////////////////////////////////////////////////////////////
 
-var WindowsFrameworks = new string[] {
-    "net-4.5", "net-4.0", "net-3.5", "net-2.0", "netstandard13", "netstandard16" };
-
-var LinuxFrameworks = new string[] {
-    "net-4.5", "net-4.0", "net-3.5", "net-2.0" };
-
-var AllFrameworks = IsRunningOnWindows() ? WindowsFrameworks : LinuxFrameworks;
+var AllFrameworks = new string[] {
+    "net45", "net40", "net35", "net20", "netstandard1.6", "netcoreapp1.1" };
```

Comment:
Do we need to maintain this list of frameworks anymore? Or could we just iterate through all the folders in `BIN_DIR`? This way is belt and braces, I guess, but extra maintenance.